src/main/kotlin/logic/exceptions/Exceptions.kt

```diff
@@ -17,7 +17,7 @@ class UserNotFoundException(message: String) : Exception(message)
 class AccessDeniedException(message: String) : Exception(message)
 class UserAlreadyExistsException(message: String) : Exception(message)
 class UnknownException(message: String) : Exception(message)
-class NoChangeHistoryFoundException(message: String) : Exception(message)
 class UnauthorizedAccessException(message: String) : Exception(message)
 class InvalidProjectInfo(message: String) : Exception(message)
-class TaskNotFoundException(message: String) : Exception(message)
\ No newline at end of file
+class TaskNotFoundException(message: String) : Exception(message)
+class UserNotFoundBySelectedUserIdException(message: String) : Exception(message)
```

Comment:
It's enough to name it:

```suggestion
class UserNotFoundException(message: String) : Exception(message)
```

Reply:
@Mohammed-Alhams 
I told here and she said "I want to make it specific " 😒